.circleci/config.yml

```diff
@@ -61,9 +64,24 @@ commands:
           path: test_root/log
       - store_test_results:
           path: reports
+      - run:
+          name: Coverage Report
+          command: |
+            pr_no=$(echo $CIRCLE_PULL_REQUEST | rev | cut -d/ -f1 | rev)
+            pr_api_url=https://api.github.com/repos/"$CIRCLE_PROJECT_USERNAME"/"$CIRCLE_PROJECT_REPONAME"/pulls/"$pr_no"
+            BASE_BRANCH=$(curl "$pr_api_url" | jq -r ".base.ref")
+            BASE_BRANCH="origin/${BASE_BRANCH}"
```

Comment:
Let's simply do `BASE_BRANCH="origin/$(curl "$pr_api_url" | jq -r ".base.ref")"`

Reply:
This resulted in issue, BTW commented this code as this will be fixed later.

Reply:
You just need to escape the inner double quotes using `\`